modules/nw-configure-ingress-access-logging.adoc

```diff
@@ -116,3 +116,47 @@ spec:
   logging:
     access: null
 ----
+
+Allowing IngressController to Modify the HAProxy Log Length when using a Sidecar
+
+* Configure `httpLogFormat` within IngressController. Leaving it blank will default to standard http log format. 
+Use `spec.logging.access.destination.syslog.maxLength` if you're using `spec.logging.access.destination.type: Syslog`.
+
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  replicas: 2
+  logging:
+    access:
+      destination:
+        type: Syslog
+        syslog:
+          address: 1.2.3.4
+          maxLength: 4096
+          port: 10514
+----
+Use `spec.logging.access.destination.container.maxLength` if you're using `spec.logging.access.destination.type: Container`.
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  replicas: 2
+  logging:
+    access:
+      destination:
+        type: Container
+        container:
+          maxLength: 4096
```

Comment:
maxLength here should be 8192, thanks.

Reply:
@ShudiLi is that for both examples?

Reply:
@jmanthei  Sorry for the late reply. Just for the Container, for the syslog, it is still 4096.